shenyu-client/shenyu-client-dubbo/shenyu-client-alibaba-dubbo/src/main/java/org/apache/shenyu/client/alibaba/dubbo/AlibabaDubboServiceBeanListener.java

```diff
@@ -84,7 +84,8 @@ public AlibabaDubboServiceBeanListener(final ShenyuRegisterCenterConfig config,
 
     private void handler(final ServiceBean<?> serviceBean) {
         Class<?> clazz = serviceBean.getRef().getClass();
```

Comment:
I think just call `Class<?> clazz = AopUtils.getTargetClass(serviceBean.getRef());` is enough. Can you try it?

Reply:
I will try it

Reply:
Any progress? @JemyXiao